ethportal-peertest/src/jsonrpc.rs

```diff
@@ -22,23 +22,13 @@ use trin_core::{
 const DATA_RADIUS: Distance = Distance::MAX;
 /// Default enr seq value
 const ENR_SEQ: &str = "1";
-/// Default history header content key presuming chain ID 3
+/// History header content key & value for merge block on mainnet
+/// Use merge block here so that we can use the default master accumulator for validation
+/// rather than needing to go lookup an epoch accumulator.
```

Comment:
Hm, don't we also want to test that validation works when looking up an old epoch? Even if we do something like drop the old epoch into the local data store, first, I'd feel better that we're testing that code path.

Reply:
Those scenarios are all unit-tested in `accumulator.rs`. I didn't feel like they needed to be replicated in `peertest` - where the test (imo) is more related to making sure the jsonrpc request/response happens successfully, and less about covering the edge cases.